setup.py

```diff
@@ -301,8 +301,10 @@ def make_release_tree(self, base_dir, files):
     ],
 
     package_dir={"": "lib"},
-    packages=find_packages("lib"),
-    namespace_packages=["mpl_toolkits"],
+    packages=find_namespace_packages(
```

Comment:
Not sure I read https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-simple-packages correctly, but could we use `find_packages(…, include=..)`? The exclude has the danger of unintentionally picking up other files we add in the future.

Reply:
I experimented a bit with it and so far I haven't been able to build matplotlib by supplying `include` instead of `exclude`. Even building an exhaustive list of all modules that should be included doesn't appear to work. I don't know if that's a bug in setuptools or if there's something I don't understand about this function.

Reply:
I think the key difference between `include` and `exclude` are a) liklihood of adding things that we want to go in each list. and b) ease of drawing the lines around the things we want included/excluded.

for a), I think it is more likely that we add python modules than folders we wish to exclude from being importable python modules (both could happen, but I'd expect python modules to be more common), so excludes are less likely to be updated

for b) I think that the things excluded appear directly next to things that want to be included, so I think it is much easier to specify exclusions.

I mean, it is just a list of strings at the end of the day, so if there is an easier way to produce the list we want, I'm not opposed, but lean towards using the tools provided to produce the lists.